README.adoc

```diff
@@ -165,7 +165,8 @@ $ make test-unit
 
 integration-tests::
 Integration tests demand more setup (i.e. the PostgreSQL DB must be already
-running) and probably time.
+running) and probably time. Please make sure you have installed psql and
+it is running on 5432 or if you have docker compose just do `docker-compose up -d`
```

Comment:
It's better to say `docker-compose up -d postgres` to just start the DB. I know there's not yet more in the compose file but there will be soon.